tgui/docs/component-reference.md

```diff
@@ -27,7 +27,6 @@ Make sure to add new items to this list if you document new components.
   - [`Flex.Item`](#flexitem)
   - [`Grid`](#grid)
   - [`Grid.Column`](#gridcolumn)
-  - [`HealthStat`](#HealthStat)
```

Comment:
eh, i kinda like this here even if it's not a proper new component, since it encourages knowledge of it and usability

Reply:
Yeah, taking out documentation felt bad. I might go add another section for common interface documentation?

At the moment we seem to be having `components` be as close to tgstation's as we can (which will make merging from upstream easier), with our "business logic" stuff in `interfaces`. We have `BeakerContents`, (now) `HealthStat`, `PortableAtmos`, and `ReleaseValve` in there now, which I think is good and logically grouped.

Will think over best way to document those and move this documentation accordingly.